dockerfiles/wolfi-base/Dockerfile

```diff
@@ -6,7 +6,6 @@ COPY ./scripts/install-wolfi-tesseract.sh install-wolfi-tesseract.sh
 
 USER root
 RUN apk update && \
-    apk del python-3.12 && \
     apk add py3.11-pip mesa-gl glib cmake bash libmagic wget git openjpeg && \
```

Comment:
should this be py3.12-pip? @MthwRobinson

Reply:
Added `py3.12-pip` in there. Figured we can keep `py3.11-pip` in there now as well so we can still upgrade 3.11 if 3.12 doesn't work for whatever reason

Reply:
Looks like you can only have either one or the other ... 3.12 it is!

Reply:
> Added py3.12-pip in there.

@MthwRobinson Does this mean we don't need to explicitly add `py3.12-pip` like `py3.11-pip`?